dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -697,4 +697,5 @@
     <!-- Tell CreateLCGFile to use the LCE file from RC.EXE -->
     <LCEFile>$(_LceFile)</LCEFile>
   </PropertyGroup>
+  <Import Project="$(MSBuildProjectDirectory)\..\..\Build\PGO\Microsoft.UI.Xaml.PGO.targets" />
```

Comment:
Is there a reason that this is at the end, instead of at the top with the rest of the Imports?